test/position-managers/IncreaseLiquidity.t.sol

```diff
@@ -164,14 +165,19 @@ contract IncreaseLiquidityTest is Test, PosmTestSetup, Fuzzers {
         uint256 balance0BeforeAlice = currency0.balanceOf(alice);
         uint256 balance1BeforeAlice = currency1.balanceOf(alice);
 
-        vm.startPrank(alice);
-        increaseLiquidity(tokenIdAlice, config, liquidityDelta, ZERO_BYTES);
-        vm.stopPrank();
+        Plan memory planner = Planner.init();
+        planner.add(Actions.INCREASE_LIQUIDITY, abi.encode(tokenIdAlice, config, liquidityDelta, ZERO_BYTES));
+        planner.add(Actions.CLEAR, abi.encode(config.poolKey.currency0, 1 wei)); // alice is willing to forfeit 1 wei
+        planner.add(Actions.CLEAR, abi.encode(config.poolKey.currency1, 1 wei));
+        bytes memory calls = planner.encode();
 
-        // alice barely spent any tokens
```

Comment:
also can we leave this test with take, and just add the clear one as well

so we have:

test_increaseLiquidity_withExactFees_take
test_increaseLiquidity_withExactFees_clear